moveit_ros/moveit_servo/include/moveit_servo/servo_parameters.h

```diff
@@ -43,6 +43,7 @@
 #include <vector>
 
 #include <rclcpp/rclcpp.hpp>
+#include <moveit/planning_scene_monitor/planning_scene_monitor.h>
```

Comment:
Nitpick: please alphabetize. I think the order should be: system includes, then MoveIt includes.

```
#include <mutex>
#include <thread>
#include <vector>

#include <moveit/planning_scene_monitor/planning_scene_monitor.h>
#include <rclcpp/rclcpp.hpp>
```